ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -40,6 +40,13 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: installer-rehearse-aws-qe-capi
```

Comment:
No need to add new rehearse test item, just replace the above line 42 with your testing workflow.

Reply:
made the change and squashed commit PTAL